misk/hibernate/src/main/kotlin/misk/hibernate/SchemaMigrator.kt

```diff
@@ -37,10 +36,7 @@ internal class SchemaMigrator(
 ) {
   /** Returns a map from version to path. */
   fun availableMigrations(): NavigableMap<Int, String> {
-    if (config.migrations_path == null) {
-      return TreeMap()
-    }
-    val resources = resourceLoader.list(config.migrations_path)
```

Comment:
I've refactored this as smart casts do not work across packages (an alternative would be `val resources = resourceLoader.list(config.migrations_path!!`)